### PR TITLE
Add a page describing the environment variables used by the CLI.

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Document AWS env vars used by `convox install` and `convox uninstall`](https://trello.com/c/WsmHKe5m/799-document-aws-env-vars-used-by-convox-install-and-convox-uninstall)

File: _reference/cli-environment-variables.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Convox pays attention to the following environment variables.
+
+## `AWS_REGION`, `AWS_DEFAULT_REGION`
+
+Amazon region in which new Racks will be installed.
+
+## `AWS_ENDPOINT`
+
+Custom Amazon endpoint.
```
Comment (mattmanning) [inline _reference/cli-environment-variables.md]: Can you elaborate on what this means?

File: _reference/cli-environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN`
+
+AWS credentials.
+
+## `CONVOX_CONFIG`
+
+Location of convox config directory.
```
Comment (mattmanning) [inline _reference/cli-environment-variables.md]: Capitalize "convox". What goes in the config directory?

File: _reference/cli-environment-variables.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+`List<AWS::EC2::Subnet::Id>`
+
+If `SUBNETS_PRIVATE` is empty, `SUBNETS` will be used.
+
+## `VERSION`
+
+Custom Rack version. Used when [installing a new Rack](/docs/installing-a-rack). Defaults to `latest`.
+
+## `TEMPLATE_FILE`
```
Comment (mattmanning) [inline _reference/cli-environment-variables.md]: This should go above `VERSION` to be alphabetical like the rest of the list.

File: _reference/cli-environment-variables.md (hunk 1 above)
Comment (soulshake) [inline _reference/cli-environment-variables.md]: I'm a bit confused as to what this envvar actually does. The only reference I found to it in the docs was [Supported AWS regions](https://convox.com/docs/supported-aws-regions/). But is that even related to the `AWS_ENDPOINT` referenced in [Rack](https://github.com/convox/rack/search?utf8=%E2%9C%93&q=AWS_ENDPOINT) (which seems like more of a global setting than a service-specific one)?

If so, would a link to http://docs.aws.amazon.com/general/latest/gr/rande.html suffice?

File: _reference/cli-environment-variables.md (hunk 2 above)
Comment (soulshake) [inline _reference/cli-environment-variables.md]: I'd like to address the config directory and its contents in another PR (ASAP). TBD today?

File: _reference/cli-environment-variables.md (hunk 3 above)
Comment (soulshake) [inline _reference/cli-environment-variables.md]: Fixed, thanks

**Pull request merged**